recipes/julia-versionparsing/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "julia-versionparsing" %}
+{% set version = "1.3.0" %}
+{% set sha256 = "f90fe419e1a40ef0eccfaaed1d1b7792d9115a059a82d0c23e3c04c944d0f8ca" %}
+{% set julia_pkg_name = "VersionParsing" %}
+{% set julia_pkg_uuid = "81def892-9a0e-5fdd-b105-ffc91e053289" %}
+{% set julia_pkg_github = "https://github.com/JuliaInterop/VersionParsing.jl" %}
+{% set julia_pkg_git = julia_pkg_github + ".git" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: {{ julia_pkg_github }}/archive/refs/tags/v{{ version }}.tar.gz 
+  sha256: {{ sha256 }}
+  # Clone the git repository rather than the tarball so Julia can use the git information
+  #git_url: {{ julia_pkg_git }}
+  #git_rev: v{{ version }}
+  #git_depth: 1
+  folder: {{ julia_pkg_name }}.jl
+
+build:
+  skip: true  # [win]
+  number: 0
+  script_env:
+    - JULIA_PKG_NAME={{ julia_pkg_name }}
+    - JULIA_PKG_UUID={{ julia_pkg_uuid }}
+    - JULIA_PKG_GIT={{ julia_pkg_git }}
+
+test:
+  requires:
+    - julia
+  commands:
+    - julia -e 'using {{ julia_pkg_name }}, Pkg; Pkg.test("{{ julia_pkg_name }}")'
+
+outputs:
+  - name: {{ name }}
+    script: build_base.sh
+    build:
+      noarch: generic
+    requirements:
+      build:
+        - julia
+        - sed
+      host:
+        - julia
+      run:
+        - julia
+    test:
+      commands:
+        - julia -e 'using {{ julia_pkg_name }}, Pkg; Pkg.test("{{ julia_pkg_name }}")'
+  - name: {{ julia_pkg_name|lower }}.jl
```

Comment:
Can we please use only one name? Planning to have alias is not a good idea. We use alias when there's confusion, but here in this case I feel like we add to the confusion by having two names.

Reply:
@ngam had the idea of using two names.

My preference is `versionparser.jl`, although `versionparser.jl-feedstock` does seem a bit strange.

Reply:
My preference is `versionparser.jl` and stick with only one name.